trunk/SUAVE/Methods/Weights/Buildups/Common/wing.py

```diff
@@ -85,22 +90,30 @@ def wing(wing,
             weight:                 Wing Mass                           [kg]
     """
 
-#-------------------------------------------------------------------------------
-# Unpack Inputs
-#-------------------------------------------------------------------------------
+    #-------------------------------------------------------------------------------
+    # Unpack Inputs
+    #-------------------------------------------------------------------------------
 
     MTOW                        = config.mass_properties.max_takeoff
     wingspan                    = wing.spans.projected
-    chord                       = wing.chords.mean_aerodynamic,
-    thicknessToChord            = wing.thickness_to_chord, 
-    wingletFraction             = wing.winglet_fraction, 
+    chord                       = wing.chords.mean_aerodynamic
+    thicknessToChord            = wing.thickness_to_chord
     wingArea                    = wing.areas.reference
 
+    try:
+        wingletFraction = wing.winglet_fraction
+    except AttributeError:
+        wingletFraction = 0.0
+
     totalWingArea = 0
     for w in config.wings:
         totalWingArea += w.areas.reference
-    liftFraction                = wingArea/totalWingArea
-    motor_spanwise_locations    = wing.motor_spanwise_locations
+    liftFraction                    = wingArea/totalWingArea
+
+    try:
+        motor_spanwise_locations    = wing.motor_spanwise_locations
```

Comment:
Can you add this to the inputs in the doc string?

Reply:
added